pkg/services/librarypanels/database.go

```diff
@@ -62,3 +62,54 @@ func (lps *LibraryPanelService) deleteLibraryPanel(c *models.ReqContext, panelID
 
 	return err
 }
+
+// getLibraryPanels gets Library Panels based on statement and args
+func getLibraryPanels(session *sqlstore.DBSession, statement interface{}, args ...interface{}) ([]LibraryPanel, error) {
+	libraryPanels := make([]LibraryPanel, 0)
+
+	err := session.SQL(statement, args...).Find(&libraryPanels)
```

Comment:
I don't see the reason for so much whitespace :)
```suggestion
	libraryPanels := make([]LibraryPanel, 0)
	err := session.SQL(statement, args...).Find(&libraryPanels)
```

Reply:
Sure I agree, will fix this in my next commit.

Reply:
Removed the whole function